sinker/heartbeats.go

```diff
@@ -22,7 +22,7 @@ const (
 
 func (svc *sinkerService) sendSingleHeartbeat(t time.Time) {
```

Comment:
I don't remember/understand the heartbeating here. I understand it's a way to try to track idle Sinks... but why is it sending redis events? And does it actually work, when the sinker service scales horizontally to multiple services?

Should we take the time now to change this to the way Agents work - a database field tracking last write, and a task which updates to idle based on that?